Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -222,9 +222,9 @@ void _delayedEndFigureCall() {
     // closed.
     bool m_hasGeometryStarted;
     // True if this geometry can end its own figures.
-    bool m_allowsFigureCalls;
+    int m_allowsFigureCalls;
```

Comment:
>int m_allowsFigureCalls [](start = 4, length = 23)

also, what exactly is this cleaning up?  you are going from bool to tri-state, why?